table/table.go

```diff
@@ -81,7 +81,7 @@ func (table *Table) GetSpoolDir() string {
 func (table *Table) Dispatch(buf []byte, val float64, ts uint32) {
 	buf_copy := make([]byte, len(buf))
 	copy(buf_copy, buf)
-	log.Debug("table received packet %s", buf_copy)
+	//log.Debug("table received packet %s", buf_copy)
```

Comment:
oops i need to back these out. This one and the log.info("table sending to route") made it difficult to debug. Actually I think making the log.info() on line 113 a log.debug() would improve the situation. It would make info mode usable, whereas right now it logs far too much under load.